assemblies/assembly_decision-manager/main.adoc

```diff
@@ -10,11 +10,10 @@ include::_artifacts/document-attributes.adoc[]
 include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
-As a business analyst or rules developer, you can use {CENTRAL} in {PRODUCT} to design a variety of decision services. In this tutorial, you will create and test a driver's license suspension scenario.
-//by creating data objects with data fields, such as Violation Date, Fine Amount, and so on.
+As a business rules developer, you can use {CENTRAL} in {PRODUCT} to design a variety of decision services. This document describes how to create and test a driver's license suspension scenario.
 
 .Prerequisites
-* {EAP_LONG} 7.1.0 is installed. For details, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/installation_guide/[_Red Hat JBoss EAP 7.1.0 Installation Guide_].
+* {EAP_LONG} {PRODUCT_VERSION} is installed. For installation information, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/installation_guide/[_Red Hat JBoss EAP 7.2.0 Installation Guide_].
```

Comment:
Wrong version variable. You want {EAP_LONG} {EAP_VERSION}, per Emily's new variable she added. Check the document-attributes.adoc for it to confirm. Pretty sure. You might have to rebase. Either way, don't use {PRODUCT_VERSION} for that.

Reply:
Done